cluster/manifests/skipper/deployment.yaml

```diff
@@ -1,7 +1,7 @@
 {{/* image-updater-bot detects *image variables so use __ suffux to disable it for main image */}}
 
-{{ $main_image__ := "container-registry.zalando.net/teapot/skipper-internal:v0.21.198-1017" }}
-{{ $canary_image := "container-registry.zalando.net/teapot/skipper-internal:v0.21.198-1017" }}
+{{ $main_image__ := "container-registry.zalando.net/teapot/skipper-internal:v0.21.208-1027" }}
+{{ $canary_image := "container-registry.zalando.net/teapot/skipper-internal:v0.21.208-1027" }}
```

Comment:
1. bot did not provide a description 
2. bot replaced both image strings since they currently match

Reply:
3. bot does not rebase PRs